### PR TITLE
feat(echo-server): add support for ingressClassName and update default app version. Fixes #39.

### LINKED ISSUE #39: [FEATURE/echo-server]
implement ingress class name

### PR DESCRIPTION
**Pull request by acobaugh**

### CODE REVIEW

File: charts/echo-server/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 appVersion: "0.4.0"
```
Comment (Ealenn) [inline charts/echo-server/Chart.yaml]: Hello @acobaugh Thanks for your PR :) Can you take the opportunity to update the application (0.5.1) ?

I don't know if we have to change something. Maybe it's better to do it in another PR. I let you see. Isn't mandatory

**Comment by acobaugh** [comment]
Updated the default version of echo-server. I don't mind rolling all of this into one PR.
